projects/admin/src/app/acquisition/components/account/account-list/account-list.component.ts

```diff
@@ -63,4 +101,57 @@ export class AccountListComponent implements OnInit {
     this.rootAccounts = this.rootAccounts.filter(item => item.pid !== account.pid);
   }
 
+  /**
+   * Get Export formats for the current resource given by configuration.
+   * @return Array of export format to generate an `export as` button or an empty array.
+   */
+  private _exportFormats(): Array<any> {
+    if (exportFormats) {
+      return exportFormats.map(
+        (format) => {
+          return {
+            label: format.label,
+            url: this.getExportFormatUrl(format),
+            disabled: !this.canExport(format),
+            disabled_message: this.exportInfoMessage
+          };
+        }
+      );
+    }
+    return [];
+  }
+
+  /**
+   * Get export format url.
+   * @param format - export format object
+   * @return formatted url for an export format.
+   */
+  getExportFormatUrl(format: any) {
+    const defaultQueryParams = [
+      'is_active:true',
+      `library.pid:${this._libraryPid}`
+    ];
+    const query = defaultQueryParams.join(' AND ')
+    const baseUrl = format.endpoint
+      ? format.endpoint
+      : this._apiService.getEndpointByType('acq_accounts');
+    let url = `${baseUrl}?q=${encodeURIComponent(query)}&format=${format.format}`;
+    // check if max rest result size is disabled
+    if (!format.disableMaxRestResultsSize) {
+      url += `&size=${RecordService.MAX_REST_RESULTS_SIZE}`;
+    }
```

Comment:
I should be prettier to use `HttpParams` to build a list of queryParams and build the query url params based ton this object. I'm not fan creating URL by hands.

Reply:
ok